### DISCUSSION TITLE
Pre(post)processing the commands of tinymce table plugin

### DISCUSSION THREAD

**Discussion by NavasardianMichael** [tinymce/tinymce#10048] [General]
Isn't there a way to preprocess the selected content of table API, before pasting, when interacting with internally generated buttons of table plugin (e.g. "paste column before" or "paste row after").
P.S. "paste_preprocess" is not fired in the mentioned scenario.
Thanks in advance!

**Comment by NavasardianMichael** [comment]
v6.7.2

**Comment by lostkeys** [comment]
Hi @NavasardianMichael, I'll convert this to a discussion topic since it isn't a bug or feature request.

**Reply by NavasardianMichael** [reply]
Sure, thank you!
